### DISCUSSION TITLE
How to specify grid size

### DISCUSSION THREAD

**Discussion by shun126** [shun126/UE5-DungeonGeneratorDemo#16] [Ideas]
How about being able to specify grid sizes separately for horizontal and vertical directions?

グリッドのサイズを水平方向と垂直方向で分けて指定できるのはどうだろう？

**Comment by erox455** [comment]
Sounds nice! :) I would like everything that makes the system more flexible.

**Reply by shun126** [reply]
Work carefully because of many changes.

**Comment by shun126** [comment]
Support completed in version 1.6.0

version 1.6.0 にて対応完了
